src/blocks/content-helper/content-helper-error.tsx

```diff
@@ -26,14 +26,14 @@ export enum ContentHelperErrorCode {
  * @see https://github.com/microsoft/TypeScript/wiki/FAQ#why-doesnt-extending-built-ins-like-error-array-and-map-work
  */
 export class ContentHelperError extends Error {
-	protected code: string;
+	protected code: ContentHelperErrorCode;
```

Comment:
Used more stricter typings.

Reply:
Asking because we have code like this:

https://github.com/Parsely/wp-parsely/blob/cea8d2418823feaf7ee3bf566c06fefe9ec13933/src/blocks/content-helper/current-post-details/provider.ts#L110-L113

`wpError.code` can be something that is not within our enumeration.

Reply:
During development we can only pass the strings defined in the enum.

But if the code is running in browser then it can work with any string because mainly TS types are for development purposes.

Having stricter types is better because in future if we pass anything to the function which isn't defined in the `enum` then we can take appropriate action.

Reply:
Thank you!